softwarereview_author.Rmd

```diff
@@ -18,7 +18,8 @@ This concise guide presents the software peer review process for you as a packag
 -   Do not submit your package for review while it or an associated manuscript is also under review at another venue, as this may result on conflicting requests for changes.
 -   Next, [open a new issue](https://github.com/ropensci/software-review/issues/new) in
 the software review repository and fill out the template.
--   Communication between authors, reviewers and editors will first and foremost take place on GitHub, although you can choose to contact the editor by email or Slack for certain issues. *When submitting a package please make sure your GitHub notification settings make it unlikely you will miss a comment.*
+-   Communication between authors, reviewers and editors will first and foremost take place on GitHub, although you can choose to contact the editor by email or Slack for certain issues. Do not contact reviewers off-thread without asking them in the GitHub thread whether they agree to it.
```

Comment:
```suggestion
-   Communication between authors, reviewers and editors will first and foremost take place on GitHub so that the review thread can serve as a full record of the review. You may choose to contact the editor by email or Slack if private consultation is needed (e.g., asking how to respond to a reviewer question). Do not contact reviewers off-thread without asking them in the GitHub thread whether they agree to it.
```